circle.yml

```diff
@@ -0,0 +1,62 @@
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ORGANIZATION
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+general:
+  # Don't run CI for PR, only for major branches
+  branches:
+    only:
+      - master
+      - /v[0-9]+(\.[0-9]+)*/
+      - feature/circleci
```

Comment:
makes sense, yeah, but I guess we should still keep `deployment.branch.only` since we likely don't want a new package on bintray for every PR.